tests/components/test_auto_fill.py

```diff
@@ -52,3 +66,19 @@ def test_auto_fill_exit_flow(kwik_e_mart_app, kwik_e_mart_app_path):
     directives = convo.process("exit").directives
     assert_reply(directives, "Sorry I cannot help you. Please try again.")
     assert_target_dialogue_state(convo, None)
+
+
+@pytest.mark.conversation
+@pytest.mark.asyncio
+async def test_auto_fill_async(async_kwik_e_mart_app, kwik_e_mart_app_path):
+    """Tests auto fill for async apps."""
+    convo = Conversation(app=async_kwik_e_mart_app, app_path=kwik_e_mart_app_path)
```

Comment:
I think if you set `app`, you don't need to set `app_path`. Setting `force_sync` equal to true will allow you to make the assertion normally without await.
 
```suggestion
    convo = Conversation(app=async_kwik_e_mart_app, force_sync=True)
```